galley/pkg/source/fs/source.go

```diff
@@ -105,8 +105,8 @@ func (s *source) readFiles(root string) map[resource.FullName]*fileResource {
 	return results
 }
 
-func (s *source) readFile(path string, info os.FileInfo, initial bool) map[resource.FullName]*fileResource {
-	result := map[resource.FullName]*fileResource{}
+func (s *source) readFile(path string, info os.FileInfo, initial bool) *[]*fileResource {
```

Comment:
I don't think you need to return a pointer here. Just returning []*fileResource should be enough.

Reply:
Sure, done